### DISCUSSION TITLE
My PC gets extremely laggy when changing the checkpoint

### DISCUSSION THREAD

**Discussion by gabriel-oj** [lllyasviel/Fooocus#3569] [Q&A]
Guys,

Whenever I decide to change the checkpoint from my default model to any other one to generate images (during Fooocus' execution), my PC gets extremely laggy as soon as I hit generate, meaning I can't even move my mouse cursor before Fooocus finishes loading the new one.

Is this normal? Is there a way to fix this?

My setup:
i5-12400F
16GB RAM DDR5
RTX 4060 Ti 8GB

I'm guessing this happens because of my low VRAM and/or RAM.

When I decide to use the checkpoint that is loaded by default, this doesn't happen.

I'm not using any arguments.

Thanks.

**Comment by IPv6** [comment]
Is this on windows? If yes than it most probably a swap problem, disk access can be slow - especially if not SSD

**Reply by gabriel-oj** [reply]
Yes, windows 11. I have a pretty good SSD, Gen4 nvme (WD SN850), that's why I was thinking this could be an issue due to low RAM / VRAM. I tried increasing virtual memory by a lot (40 - 45GB), but it didn't help. I leave it on automatic by default.
